packages/edit-site/src/components/sidebar/global-styles/palette.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalItemGroup as ItemGroup,
+	FlexItem,
+	__experimentalHStack as HStack,
+	__experimentalZStack as ZStack,
+	__experimentalVStack as VStack,
+	FlexBlock,
+	ColorIndicator,
+} from '@wordpress/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Subtitle from './subtitle';
+import { useSetting } from '../../editor/utils';
+import NavigationButton from './navigation-button';
+
+function Palette( { contextName } ) {
+	const colors = useSetting( 'color.palette', contextName );
+	const screenPath =
+		contextName === 'root'
+			? '/colors/palette'
+			: '/blocks/' + contextName + '/colors/palette';
```

Comment:
It may be a good idea to move the path computation logic to a separate file for all screens. This would also avoid the risk of introducing typos in the paths

Reply:
I agree, I think the whole sidebar components need to be better organized, I'll do it in a separate PR.